### PR TITLE
CRYP-56: Add Models For Db Collections

### PR DESCRIPTION
**Pull request by Afaqsoftaxus**
## Jira

https://softaxus-jira.atlassian.net/browse/CRYP-56

## Changes

- Added models for DB collections in domain project

## Testing

- This ticket doesn't requires testing

### CODE REVIEW

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    public string EventName { get; set; }
+
+    [BsonElement(elementName: "buyer", Order = 10)]
+    public string? Buyer { get; set; } 
+    
+    [BsonElement(elementName: "price", Order = 11)]
+    public decimal Price{ get; set; }
+
+    [BsonElement(elementName: "offerAmount", Order = 12)]
+    public decimal? OfferAmount { get; set; }
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AllEventDocument .cs:37]: Remove nullable

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
+﻿namespace CryptoAxus.Domain.Collections;
+
+[BsonCollection("AllEvent")]
+public class AllEventDocument: BaseDocument
+{
+    [BsonElement(elementName: "nftId", Order = 2)]
+    public string NftId { get; set; }
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AllEventDocument .cs:7]: Check with the frontend team which fields are nullable and place the relevant check on it.

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    public bool  IsSold { get; set; }
+
+    [BsonElement(elementName: "accepted", Order = 18)]
+    public bool Accepted { get; set; }
+
+    [BsonElement(elementName: "rejected", Order = 19)]
+    public bool Rejected { get; set; }
+
+
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AllEventDocument .cs:61]: Remove these unnecessary spaces.

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    public AllEventDocument() : base()
+    {
+    }
+    public AllEventDocument(ObjectId id,
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AllEventDocument .cs:65]: Always leave a single line empty before start of the constructor

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                DateTime? canceledAt = null,
+                DateTime? soldAt = null,
+                String? walletAddress = null,
+                DateTime? lastModifiedDate = null,
+                ObjectId? lastModifiedBy = null)
+                          : base(id,
+                                 createdBy,
+                                 lastModifiedDate,
+                                 lastModifiedBy,
+                                 isDeleted)
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AllEventDocument .cs:92]: Align the code properly. For reference see the `ArtistDocument` model.

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    {
+         Id = id;
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AllEventDocument .cs:94]: Fix the alignment

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+        SoldAt = soldAt;
+        IsSold = isSold;
+        Accepted = accepted;
+        Rejected = rejected;
+        CreatedBy = createdBy;
+        LastModifiedDate = lastModifiedDate;
+        LastModifiedBy = lastModifiedBy;
+        IsDeleted = isDeleted;
+    
+}
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AllEventDocument .cs:118]: 1. Remove unnecessary spaces.
2. Make sure the indentation is correct.

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+    [BsonElement(elementName: "softDelete", Order = 6)]
+    public bool SoftDelete { get; set; }
+
+    [BsonElement(elementName: "startingDateTime", Order = 7)]
+    public DateTime StartingDateTime { get; set; }
+    
+    [BsonElement(elementName: "endingDateTime", Order = 8)]
+    public DateTime EndingDateTime { get; set; } 
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AuctionDocument .cs:26]: As mentioned above remove extra spaces

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+                          ObjectId createdBy,
+                          DateTime? lastModifiedDate = null,
+                          ObjectId? lastModifiedBy = null,
+                          DateTime? deleted_at = null,
+                          bool isDeleted = false)
+                          : base(id,
+                                 createdBy,
+                                 lastModifiedDate,
+                                 lastModifiedBy)
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AuctionDocument .cs:49]: Align code and remove extra spaces

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+        SoftDelete = softDelete;
+        StartingDateTime = startingDateTime;
+        CreatedBy = createdBy;
+        LastModifiedDate = lastModifiedDate;
+        LastModifiedBy = lastModifiedBy;
+        IsDeleted = isDeleted;
+    
+}
+}
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AuctionDocument .cs:65]: As discussed above.

File: src/CryptoAxus.Domain/Collections/BiddingDocument .cs [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+        LastModifiedDate = lastModifiedDate;
+        LastModifiedBy = lastModifiedBy;
+        IsDeleted = isDeleted;
+    
+
+
+}
+}
+
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/BiddingDocument .cs:64]: As discussed above

File: src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+    {
+    }
+
+    public FixedPricedDocument(ObjectId id,
+                          string nftCopyId,
+                          int quantity,
+                          decimal pricePerItem,
+                          DateTime fixedEndingDateTime,
+                          bool softDelete,
+                         //ateTime? deleted_at = null,
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs:31]: Remove commented code.

File: src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+    {
+
+        Id = id;
+        NftId = nftId;
+        ArtistWalletAddress = artistWalletAddress;
+        CollectionId = collectionId;
+       
+
+    }
+}
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs:32]: Fix code alignment

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+
+    [BsonElement(elementName: "name", Order = 8)]
+    public string Name { get; set; }
+
+    [BsonElement(elementName: "url", Order = 9)]
+    public string Url { get; set; }
+
+    [BsonElement(elementName: "description", Order = 10)]
+    public string Description { get; set; }
+    [BsonElement(elementName: "collection", Order = 11)]
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:32]: Add empty line between each property

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+
+    [BsonElement(elementName: "blockchain", Order = 13)]
+    public string Blockchain { get; set; }
+
+    [BsonElement(elementName: "createrEarnings", Order = 14)]
+    public int CreaterEarnings { get; set; }
+ 
+    public MainNftsDocument() : base()
+    {
+    }
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:46]: Add empty line

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+        CollectionId = collectionId;
+        Blockchain = blockchain;
+        CreaterEarnings = createrEarnings;
+        CreatedBy = createdBy;
+        LastModifiedDate = lastModifiedDate;
+        LastModifiedBy = lastModifiedBy;
+        IsDeleted = isDeleted;
+    
+}
+}
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:93]: Fix code alignment

File: src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs [hunk 17]
```diff
@@ -0,0 +1,84 @@
...
+    [BsonElement(elementName: "description", Order = 8)]
+    public string Description { get; set; }
+
+    [BsonElement(elementName: "category", Order = 9)]
+    public string Category { get; set; }
+
+    [BsonElement(elementName: "collectionBlockChain", Order = 10)]
+    public string CollectionBlockChain { get; set; }
+
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs:33]: Remove empty line

File: src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+        LastModifiedBy = lastModifiedBy;
+        IsDeleted = isDeleted;
+    
+
+
+}
+}
+
+
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs:63]: Fix code alignment and code indentation

File: src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs [hunk 19]
```diff
@@ -0,0 +1,63 @@
...
+    public int MarketTokenId { get; set; }
+
+    public NFTCopiesDocument() : base()
+    {
+    }
+
+    public NFTCopiesDocument(ObjectId id, 
+                          string nftId,
+                          string owner,
+                          int quantity,
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs:31]: No parameter for `OwnerName` field added.

File: src/CryptoAxus.Domain/Collections/OffersDocument .cs [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+        OfferPrice = offerPrice;
+        OfferFrom = offerFrom;
+        OfferTo = offerTo;
+        Quantity = quantity;
+        OfferExpireAt = offerExpireAt;
+        OfferedQuantity = offeredQuantity;
+        CreatedBy= createdBy;
+
+    }
+}
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/OffersDocument .cs:62]: Fix code alignment

File: src/CryptoAxus.Domain/CryptoAxus.Domain.csproj [hunk 21]
```diff
@@ -7,9 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Folder Include="Collections\" />
     <Folder Include="Interfaces\" />
-    <Folder Include="Attributes\" />
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/CryptoAxus.Domain.csproj:12]: Why these lines were changed?

**Review by junaid-softaxus — CHANGES_REQUESTED** [review]
Make the changes mentioned in the PR review

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs (hunk 2 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AllEventDocument .cs:7]: Resolved

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 22]
```diff
@@ -0,0 +1,119 @@
...
+    public int Quantity { get; set; }
+
+    [BsonElement(elementName: "seller", Order = 8)]
+    public string Seller { get; set; }
+
+    [BsonElement(elementName: "eventName", Order = 9)]
+    public string EventName { get; set; }
+
+    [BsonElement(elementName: "buyer", Order = 10)]
+    public string? Buyer { get; set; }
```
[the remark this answers is not in the document]
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AllEventDocument .cs:31]: Resolved

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs (hunk 3 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AllEventDocument .cs:61]: Resolved

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 23]
```diff
@@ -0,0 +1,119 @@
...
+    public bool Accepted { get; set; }
+
+    [BsonElement(elementName: "rejected", Order = 19)]
+    public bool Rejected { get; set; }
+
+
+
+    public AllEventDocument() : base()
+    {
+    }
```
[the remark this answers is not in the document]
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AllEventDocument .cs:64]: Resolved

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs (hunk 5 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AllEventDocument .cs:92]: Resolved

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs (hunk 8 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AuctionDocument .cs:26]: Resolved

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs (hunk 9 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AuctionDocument .cs:49]: Resolved

File: src/CryptoAxus.Domain/Collections/BiddingDocument .cs (hunk 11 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/BiddingDocument .cs:64]: Resolved

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs (hunk 14 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:32]: Resolved

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs (hunk 15 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:46]: Resolved

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs (hunk 16 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:93]: Resolved

File: src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs (hunk 17 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs:33]: Resolved

File: src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs (hunk 19 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs:31]: Resolved

File: src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs (hunk 18 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs:63]: Resolved

File: src/CryptoAxus.Domain/Collections/OffersDocument .cs (hunk 20 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/OffersDocument .cs:62]: Resolved

File: src/CryptoAxus.Domain/Collections/ArtistDocument.cs [hunk 24]
```diff
@@ -31,6 +31,7 @@ public class ArtistDocument : BaseDocument
     public string Twitter { get; set; }
 
     public ArtistDocument() : base()
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/ArtistDocument.cs:34]: Blank space?

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs [hunk 25]
```diff
@@ -0,0 +1,63 @@
...
+    public bool SoftDelete { get; set; }
+
+    [BsonElement(elementName: "startingDateTime", Order = 7)]
+    public DateTime StartingDateTime { get; set; }
+    
+    [BsonElement(elementName: "endingDateTime", Order = 8)]
+    public DateTime EndingDateTime { get; set; } 
+    
+    public AuctionDocument() : base()
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AuctionDocument .cs:28]: Blank space?

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs [hunk 26]
```diff
@@ -0,0 +1,63 @@
...
+                          int quantity,
+                          decimal minimumBid,
+                          bool softDelete,
+                          DateTime startingDateTime,
+                          DateTime endingDateTime,
+                          ObjectId createdBy,
+                          DateTime? lastModifiedDate = null,
+                          ObjectId? lastModifiedBy = null,
+                          DateTime? deleted_at = null,
+                          bool isDeleted = false)
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/AuctionDocument .cs:44]: Alignment

File: src/CryptoAxus.Domain/Collections/BiddingDocument .cs [hunk 27]
```diff
@@ -0,0 +1,61 @@
...
+    public string FromWalletAddress { get; set; }
+
+    [BsonElement(elementName: "bidderUserName", Order = 6)]
+    public string? BidderUserName { get; set; } 
+
+    [BsonElement(elementName: "highestBid", Order = 7)]
+    public bool HighestBid { get; set; }
+
+    public BidingDocument() : base()
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/BiddingDocument .cs:25]: Blank space?

File: src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs [hunk 28]
```diff
@@ -0,0 +1,54 @@
...
+    public decimal PricePerItem { get; set; }
+
+    [BsonElement(elementName: "fixedEndingDateTime", Order = 5)]
+    public DateTime FixedEndingDateTime { get; set; }
+
+    [BsonElement(elementName: "softDelete", Order = 7)]
+    public bool SoftDelete { get; set; }
+    
+    public FixedPricedDocument() : base()
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs:22]: Blank space?

File: src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs [hunk 29]
```diff
@@ -0,0 +1,54 @@
...
+                               string nftCopyId,
+                               int quantity,
+                               decimal pricePerItem,
+                               DateTime fixedEndingDateTime,
+                               bool softDelete,
+                               ObjectId createdBy,
+                               DateTime? lastModifiedDate = null,
+                               ObjectId? lastModifiedBy = null,
+                               bool isDeleted = false)
+                               :base(id,
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs:36]: Add space between `:` and `base`

File: src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs [hunk 30]
```diff
@@ -0,0 +1,54 @@
...
+                                     createdBy,
+                                     lastModifiedDate,
+                                     lastModifiedBy,
+                                     isDeleted)
+        {
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs:42]: Blank space?

File: src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs [hunk 31]
```diff
@@ -0,0 +1,31 @@
...
+    public string NftId { get; set; }
+
+    [BsonElement(elementName: "artistWalletAddress", Order = 3)]
+    public string ArtistWalletAddress { get; set; }
+
+    [BsonElement(elementName: "collectionId", Order = 4)]
+    public string CollectionId { get; set; }
+    
+    public FavoriteNftsDocument() : base()
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs:16]: Blank space?

File: src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs [hunk 32]
```diff
@@ -0,0 +1,31 @@
...
+    
+    {
+    }
+
+    public FavoriteNftsDocument(ObjectId id,
+                                string nftId,
+                                string artistWalletAddress,
+                                string collectionId)
+    {
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs:25]: Blank space?

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+                            string collection,
+                            string collectionId,
+                            string blockchain,
+                            int createrEarnings,
+                            ObjectId createdBy,
+                            DateTime? lastModifiedDate = null,
+                            ObjectId? lastModifiedBy = null,
+                            string? signature = null,
+                            bool isDeleted = false)
+                            :base(id,
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:68]: Add scpace

File: src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs [hunk 34]
```diff
@@ -0,0 +1,80 @@
...
+    public string Category { get; set; }
+
+    [BsonElement(elementName: "collectionBlockChain", Order = 10)]
+    public string CollectionBlockChain { get; set; }
+
+    [BsonElement(elementName: "explicitContent", Order = 13)]
+    public bool ExplicitContent { get; set; }
+   
+    public NFTCollectionsDocument () : base()
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs:37]: Blank space?

File: src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs [hunk 35]
```diff
@@ -0,0 +1,61 @@
...
+    public int Quantity { get; set; }
+
+    [BsonElement(elementName: "isCreator", Order = 6)]
+    public bool IsCreator { get; set; }
+
+    [BsonElement(elementName: "marketTokenId", Order = 7)]
+    public int MarketTokenId { get; set; }
+
+    public NFTCopiesDocument() : base()
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs:25]: Blank space?

File: src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs [hunk 36]
```diff
@@ -0,0 +1,61 @@
...
+        Quantity = quantity;
+        IsCreator = isCreator;
+        MarketTokenId = marketTokenId;
+        CreatedBy = createdBy;
+        LastModifiedDate = lastModifiedDate;
+        LastModifiedBy = lastModifiedBy;
+        IsDeleted = isDeleted;
+    }
+}
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs:59]: Blank space?

File: src/CryptoAxus.Domain/CryptoAxus.Domain.csproj [hunk 37]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Folder Include="Collections\" />
+	<Folder Include="Collections\" />
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/CryptoAxus.Domain.csproj:10]: Alignment?

File: src/CryptoAxus.Domain/Collections/AllEventDocument .cs [hunk 38]
```diff
@@ -0,0 +1,118 @@
...
+    public bool  IsSold { get; set; }
+
+    [BsonElement(elementName: "accepted", Order = 18)]
+    public bool Accepted { get; set; }
+
+    [BsonElement(elementName: "rejected", Order = 19)]
+    public bool Rejected { get; set; }
+
+    public AllEventDocument() : base()
+
```
[the remark this answers is not in the document]
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AllEventDocument .cs:61]: Resolved

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs (hunk 25 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AuctionDocument .cs:28]: Resolved

File: src/CryptoAxus.Domain/Collections/AuctionDocument .cs (hunk 26 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/AuctionDocument .cs:44]: Resolved

File: src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs (hunk 28 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs:22]: Resolved

File: src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs (hunk 30 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/FIxedPricedDocument .cs:42]: Resolved

File: src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs (hunk 31 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs:16]: Resolved

File: src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs (hunk 32 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/FavoriteNftsDocument .cs:25]: Resolved

File: src/CryptoAxus.Domain/Collections/MainNftsDocument .cs (hunk 33 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/MainNftsDocument .cs:68]: Resolved

File: src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs (hunk 34 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs:37]: Resolved

File: src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs [hunk 39]
```diff
@@ -0,0 +1,80 @@
...
+                                   string description,
+                                   string category,
+                                   string collectionBlockChain,
+                                   bool explicitContent, 
+                                   ObjectId createdBy,
+                                   DateTime? lastModifiedDate = null,
+                                   ObjectId? lastModifiedBy = null,
+                                   string? lastUpdated = null,
+                                   bool isDeleted = false)
+                                   :base(id,
```
[the remark this answers is not in the document]
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/NFTCollectionsDocument .cs:57]: Resolved

File: src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs (hunk 36 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/Collections/NFTCopiesDocument .cs:59]: Resolved

File: src/CryptoAxus.Domain/CryptoAxus.Domain.csproj [hunk 40]
```diff
@@ -7,9 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Folder Include="Collections\" />
-    <Folder Include="Interfaces\" />
-    <Folder Include="Attributes\" />
+	  <Folder Include="Collections\" />
+      <Folder Include="Interfaces\" />
+      <Folder Include="Attributes\" />
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Domain/CryptoAxus.Domain.csproj:12]: Alignment not fixed

Comment (Afaqsoftaxus) [reply src/CryptoAxus.Domain/CryptoAxus.Domain.csproj:12]: Resolved

**Review by junaid-softaxus — APPROVED** [review]

**Pull request merged**